src/main/java/sirius/web/http/TunnelHandler.java

```diff
@@ -397,6 +394,14 @@ private void logTiming(org.asynchttpclient.HttpResponseStatus status) {
         }
     }
 
+    private String safeConvertDuration(long durationMillis) {
+        if (durationMillis <= 0) {
```

Comment:
this is a logic change. before 0 would be actually displayed as 0 and not -. Is this intended?

Reply:
Yep, as otherwise we skipped the output entirely which is sort of a bug unexpected input for NLS.formatDuration